index.js

```diff
@@ -0,0 +1,236 @@
+let playerTwoArr = []
+let winCom = []
+let clockCount = 0
+let playerOneDisplay = ""
+let playerTwoDisplay = ""
+
+//start function - when start button gets clicked
+function start() {
+    currentPlayer = 1
+    startButton.disabled = true
+
+    
+
+    //adding event listener to cell class
+    for (let cell of cells) {
+        cell.addEventListener('click', clicked)
+        cell.innerHTML = ""
+        cell.style.backgroundColor = "white"
+    }
+    //resetting arrays
+    boardArr = [0, 1, 2, 3, 4, 5, 6, 7, 8]
+    playerOneArr = []
+    playerTwoArr = []
+    winCom = []
+    //setting up clock
+    interval = setInterval(() => { updateClock() }, 1000)
+    clockCount = 0
+    clock.innerHTML = `Time Elapse: 0 seconds`
+    //two players
+    console.log(playerOneName)
+    playerOneDisplay = (playerOneName.value === "" ? "X" : playerOneName.value)
+    playerTwoDisplay = (playerTwoName.value === "" ? "O" : playerTwoName.value)
+    playerOneName.disabled = true
+    playerTwoName.disabled = true
+    showPlayer()
+}
+
+//adding event listener to start button
+startButton.addEventListener('click', start);
+
+//adding event listener to player/computer
+playerComputer.addEventListener('click', disPlayerPlayer)
+
+//funtion to disable player vs player box
+function disPlayerPlayer() {
+    playerPlayer.style.display ="none"
+    startButton.disabled = false
+    playerTwoName.style.display = "none"
+    gameMode = 'pvc' // set the gameMode to player vs computer
+    console.log(`THE GAMEMODE IS NOW: ${gameMode}`)
+}
+
+//adding event listener to player/player
+playerPlayer.addEventListener('click', disPlayerComputer)
+
+//funtion to disable player vs computer box
+function disPlayerComputer() {
+    playerComputer.style.display = "none"
+    startButton.disabled = false
+    gameMode = 'pvp' // Set the gameMode to player vs player
+    console.log(`THE GAMEMODE IS NOW: ${gameMode}`)
+}
+
+//function to update clock
+function updateClock() {
+    clockCount++
+    clock.innerHTML = `Time Elapse: ${clockCount} seconds`
+}
+
+//function to change status
+function showPlayer() {
+    if (currentPlayer === 1) {
+        gameStatus.innerHTML = `Player ${playerOneDisplay}'s turn`
+    } else {
+        gameStatus.innerHTML = `Player ${playerTwoDisplay}'s turn`
+    }
+}
+
+// function to prevent clicking on the same cell
+function alreadyClicked() {
+    gameStatus.innerHTML = `Please select an empty cell.`
+}
+
+//function to check for wins
+function checkWin(currentPlayerArr) {
+
+    let winner = false
+    wins.forEach((win) => {
+        if (currentPlayerArr.includes(win[0])) {
+            if (currentPlayerArr.includes(win[1])) {
+                if (currentPlayerArr.includes(win[2])) {
+                    winCom.push(...win)
+                    console.log(winCom)
+                    console.log("You Won!")
+                    winner = true
+                }
+            }
+        }
+    })
+    return winner
+}
```

Comment:
Great way to test for a win condition!  This explicitly states exactly what you need to.  Nesting so many `if` statements can be a good hint that you might want to consider refactoring.  In this case, for every win array, you are checking each value against the player array;  
To me, this is just asking for another iterator instead of directly accessing things by index number.  Not a big deal when the array is only 3 long, but when comparing arrays in the future, thing about iterating over BOTH to get the result you want.